ci-operator/step-registry/cucushift/upgrade/arch-migration/cucushift-upgrade-arch-migration-commands.sh

```diff
@@ -359,11 +396,25 @@ fi
 export OC="run_command_oc"
 
 SOURCE_VERSION="$(oc get clusterversion --no-headers | awk '{print $2}')"
-SOURCE_XY_VERSION="$(echo "${SOURCE_VERSION}" | cut -f1,2 -d.)"
 export SOURCE_VERSION
+SOURCE_MINOR_VERSION="$(echo "${SOURCE_VERSION}" | cut -f2 -d.)"
+export SOURCE_MINOR_VERSION
+SOURCE_XY_VERSION="$(echo "${SOURCE_VERSION}" | cut -f1,2 -d.)"
+export SOURCE_XY_VERSION
+if [[ "${SOURCE_MINOR_VERSION}" -gt 17 ]]; then
+    OLD_OPERATOR_IMAGE="$(oc get co/machine-config -oyaml | yq -r '.status.versions[] | select(.name=="operator-image") | .version')"
+    export OLD_OPERATOR_IMAGE
+fi
 
 switch_channel
 migrate
+if [[ "${SOURCE_MINOR_VERSION}" -gt 17 ]]; then
+    wait_upgrade_start
+fi
 check_migrate_status
 check_arch
-health_check
+if [[ "${SOURCE_MINOR_VERSION}" -gt 17 ]]; then
+    check_co
```

Comment:
`it looks like check_co is not the key point, right`, yes, you are right. Actually I also discussed with @lwan-wanglin , we all agree this is not necessary. I keep it here is just because this is one change in https://issues.redhat.com/browse/OTA-962.  As you and @jianlinliu both concerned with it, I will remove it.

Reply:
> From cvo side, check_migrate_status needed to be updated to catch the dummy succeed issue.

We only discussed around adding node immediately, do you still plan to fix the fake success issue in OTA-960?

Reply:
Count the progressing number in https://github.com/openshift/release/compare/5a3c8845eb403464e2155f8b9239185fbbb8745f..1fe9c93d7d855f4c3f15389b4e094a92e4876acb